build/build.go

```diff
@@ -96,8 +97,46 @@ func (d *Driver) readBuildManifest() (*buildManifest, error) {
 		// if it's not a go build and build image is not specified - use native runtime image
 		m.Native.Build.Image = m.Native.Image
 	}
+	for _, s := range m.Native.Deps {
+		if !isApkOrApt(s) {
+			return nil, fmt.Errorf("only apt/apk commands allowed in deps for final image")
+		}
+	}
 	return &m, nil
 }
+
+func isApkOrApt(s string) bool {
```

Comment:
As discussed previously, we'll restrict this at the moment, but we'll have further discussion about not restricting or restricting it further.